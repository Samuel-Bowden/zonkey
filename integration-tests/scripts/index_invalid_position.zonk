start {
	let array = Integer[1, 2, 3, 4];
	
	for (let i = 0, i < 5, i += 1) {
		println(integer_to_string(array.get(i)));
	}
}
