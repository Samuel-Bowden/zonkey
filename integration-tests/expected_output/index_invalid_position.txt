(EXCEPTION) IndexOutOfRange
  Attempted to index an array at position 4, but the array only contains 4 values.
	5 | 		println(integer_to_string(array.<get>(i)));
